docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -1,10 +1,23 @@
 [[recovery]]
 === Indices Recovery
 
-The following _expert_ setting can be set to manage the recovery policy.
+_Recovery_ is the process used to build a new copy of a shard on a node by
```

Comment:
you're giving a definition of peer recoveries here, not recoveries in general (see e.g. docs for `_cat/recovery`)

Reply:
perhaps we can crosslink to those docs for the explanation of what recoveries are